irohad/main/irohad.cpp

```diff
@@ -38,6 +38,20 @@ bool validate_config(const char *flag_name, std::string const &path) {
   return not path.empty();
 }
 
+/**
+ * Gflag validator.
+ * Validator for the blockstrore strategy.
+ * strategy is overwrite, continue or abort.
+ * @param flag_name - flag name. Must be 'keypair_name' in this case
+ * @param path      - file name. Should be path to the keypair files
+ * @return true if argument is valid
+ */
+bool validate_blockstore_strategy(const char *flag_name,
```

Comment:
Do you need `flag_name` parameter as a function argument? You should either validate it as well to be "blockstore_strategy", as I understand, or remove it

Reply:
this is Gflags api